app/assets/stylesheets/application.css.scss

```diff
@@ -24,10 +26,10 @@
 }
 
 body {
- background-image: url('http://wallpaperpassion.com/upload/13897/yoga-wallpaper.jpg');
- background-repeat:no-repeat;
- background-size: 100% 100%;
- font-family: 'Poiret One', cursive;
+  background-image: url('http://wallpaperpassion.com/upload/13897/yoga-wallpaper.jpg');
```

Comment:
Prefer double-quoted strings<br>URL `http://wallpaperpassion.com/upload/13897/yoga-wallpaper.jpg` should not contain protocol or domain